zerver/openapi/zulip.yaml

```diff
@@ -8984,33 +8988,55 @@ paths:
                 ping_only:
                   type: boolean
                   description: |
-                    If `true`, the presence information of other users will not
-                    be included in the response.
-                  example: true
-                new_user_input:
-                  type: boolean
-                  description: |
-                    Whether the user on this client has interacted with the system recently.
-                  example: true
-                slim_presence:
```

Comment:
Aside of changing phrasing to the one in my doc since I wrote a bit more detail on these there, I also reordered things to move these params below `last_update_id` in the doc. That's because they probably shouldn't even be used by someone implementing a client {new_user_input is kinda obscure, and slim_presence doesn't need to be passed if you're passing last_update_id), so let's not distract the reader by making them look at these before seeing the actual important `last_update_id` param.

Reply:
Yeah I ended up sorting them alphabetically, `last_update_id` should just be first.